js/ml/classification-label.ts

```diff
@@ -0,0 +1,120 @@
+import { LitElement, css, html } from "lit";
+import { property } from "lit/decorators.js";
+
+export class ShinyClassificationLabel extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+      padding-bottom: 2rem;
+    }
+
+    .wrapper {
+      border: 1px solid #ccc;
+      padding: 12px;
+      border-radius: 5px;
+    }
+
+    .item {
+      margin-top: 20px;
+      margin-bottom: 20px;
+    }
+
+    .winner {
+      display: flex;
+      justify-content: center;
+      font-weight: bold;
+      font-size: 1.75em;
+    }
+
+    .bar {
+      height: 5px;
+      margin-top: 5px;
+      margin-bottom: 5px;
+      background-color: #4890e3;
+      border-radius: 2px;
+    }
+
+    .label {
+      display: flex;
+      flex-direction: row;
+      align-items: baseline;
+    }
+
+    .dashed-line {
+      flex: 1 1 0%;
+      border-bottom: 1px dashed #888;
+      margin-left: 0.5em;
+      margin-right: 0.5em;
+    }
```

Comment:
Were you planning to make these rules use CSS variables? And/or default to existing CSS variables?